flink-streaming-java/src/test/java/org/apache/flink/streaming/util/TwoInputStreamOperatorTestHarness.java

```diff
@@ -64,4 +64,9 @@ public void processWatermark1(Watermark mark) throws Exception {
 	public void processWatermark2(Watermark mark) throws Exception {
 		twoInputOperator.processWatermark2(mark);
 	}
+
+	public void advanceWatermark(Watermark mark) throws Exception {
```

Comment:
`advanceWatermark` is inconsistent with `processWatermark1` and `processWatermark2`. Why haven't you called it `processWatermarks` or `processBothWatermarks`?